uint32 time_ms # Test time in milliseconds.
---
bool success # Set to true on success
float32 velocity # Found velocity in rad/s.